### PR TITLE
Make 'latest' text file publicly accessible on S3

### LINKED ISSUE #108: `latest` file should be publicly accessible
The `latest` text file that contains the name of the latest OS version should be publicly readable, enabling a simple way to programmatically retrieve the most up to date OS version.
This line needs to be changed -->
https://github.com/resin-os/resin-yocto-scripts/blob/49130dd8f1d14a33b82875d5c528fa6790df31d9/automation/jenkins_build.sh#L343

https://www.flowdock.com/app/rulemotion/resin-devices/threads/qgUIi7CftRNEbHcUfFcfEbR90Uo

### PR DESCRIPTION
**Pull request by LucianBuzzo**
Fixes #108

### CODE REVIEW

File: automation/jenkins_build.sh [hunk 1]
```diff
@@ -340,7 +340,7 @@ if [ -z "$($S3_CMD ls s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/)" ] || [ -n "$
 	$S3_CMD put /host/images/${SLUG}/${BUILD_VERSION}/IGNORE s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/
 	$S3_CMD $S3_SYNC_OPTS sync /host/images/${SLUG}/${BUILD_VERSION}/ s3://${S3_BUCKET}/${SLUG}/${BUILD_VERSION}/
 	if [ "${DEVELOPMENT_IMAGE}" = "no" ]; then
-		$S3_CMD put /host/images/${SLUG}/latest s3://${S3_BUCKET}/${SLUG}/
+		$S3_CMD put /host/images/${SLUG}/latest s3://${S3_BUCKET}/${SLUG}/ --acl-public
```
Comment (floion) [inline automation/jenkins_build.sh]: Shouldn't this be "--acl-public-read"?

Comment (floion) [reply automation/jenkins_build.sh]: Later edit: no it should not :)

**Comment by LucianBuzzo** [comment]
@floion What's the merge process on this repo as it isn't VB enabled? should we enable VB?

**Comment by floion** [comment]
We manually merge for now

**Pull request merged**